custom_visibility/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -89,11 +99,20 @@ public function addVisibilityWhere(&$query)
     }
 
     /**
-     * {@inheritdoc}
+     * orvewrite parent method, SugarVisibility::addVisibilityWhereQuery
+     *
+     * add visibility, i.e. condition, clauses to the WHERE part of SugarQuery
+     *
+     * @param SugarQuery $query
+     * @param array $options
+     * @return SugarQuery
```

Comment:
I'd add a few words to the return statement.  Perhaps something like "The updated query"